scripts/delayed.js

```diff
@@ -5,3 +5,5 @@ import { loadScript, sampleRUM } from './aem.js';
 sampleRUM('cwv');
 
 loadScript('https://icici-securities.allincall.in/files/deploy/embed_chatbot_11.js?version=1.1');
+
+loadScript('https://assets.adobedtm.com/64c36731dbac/390f7bab5b74/launch-285ee83071cc-development.min.js', { async: true });
```

Comment:
We need to add environment check to have appropriate script load. [Please refer this](https://github.com/sunstar-foundation/sunstar-foundation/blob/8c18978591b1caaef7c6b3696f671abca5c3f52e/scripts/delayed.js#L9-L16)